src/libraries/System.Reflection.Emit/src/System.Reflection.Emit.csproj

```diff
@@ -15,6 +16,7 @@
     <Compile Include="System\Reflection\Emit\PseudoCustomAttributesData.cs" />
     <Compile Include="System\Reflection\Emit\TypeBuilderImpl.cs" />
     <Compile Include="System\Reflection\Emit\SignatureHelper.cs" />
+    <Compile Include="System\Reflection\Emit\TypeNameBuilder.cs" />
```

Comment:
nit: looks like the existing TypeBuilderImpl.cs was not sorted alphabetically above.